lib/pages/course_card.dart

```diff
@@ -0,0 +1,180 @@
+import 'package:flutter/material.dart';
+
+List<CardItem> items = [
+  const CardItem(
+      image: "card1.png",
+      title: 'Art & Humanities',
+      subtitle: 'Draw & Paint Arts',
+      duration: '2Hr 20Mins'),
+  const CardItem(
+      image: "card2.png",
+      title: 'Computer & Technology',
+      subtitle: 'Programming',
+      duration: '4Hr 20Mins'),
+  const CardItem(
+      image: "card1.png",
+      title: 'Art & Humanities',
+      subtitle: 'Draw & Paint Arts',
+      duration: '2Hr 20Mins'),
+  const CardItem(
+      image: "card2.png",
+      title: 'Computer & Technology',
+      subtitle: 'Programming',
+      duration: '4Hr 20Mins'),
+];
+
+class CourseCard extends StatelessWidget {
+  const CourseCard({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return SizedBox(
+      height: 250,
+      child: ListView.builder(
+        scrollDirection: Axis.horizontal,
+        itemCount: items.length,
+        itemBuilder: ((context, index) => NewCard(item: items[index])),
+      ),
+    );
+  }
+}
+
+class CardItem {
```

Comment:
proper file structure for flutter projects.
lib/model/
lib/screens/
lib/utils/
lib/repositories

Reply:
done